src/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com");
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+:root {
+  --text-color: #060b35;
```

Comment:
It's a good practice to use more descriptive names for your variables. Consider renaming '--background-origin' to '--background-color'.